# no-blank-line-before-single-return
added in: 1.6.1 <span style={{color: 'green'}}>style</span>

Disallows empty blank lines before a return inside a code block that only contains a return statement.

## Example
### Bad:
```dart
  // LINT
  if ( ... ) {

    return;
  }
```
### Good:
```dart
  if ( ... ) {
    return;
  }

  if ( ... ) {
    // TODO(developername): need to implement.
    return;
  }
```
