# prefer-single-quotes
added in: 1.6.1 <span style={{color: 'green'}}>style</span>

Warns when strings are declared using double quotes instead of single quotes

## Example
### Bad:
```dart
...
const string = "Value"; // LINT
```

### Good:
```dart
const string = 'Value';
```
